src/gobble.py

```diff
@@ -64,49 +59,23 @@ def main():
         bus_thread.join()
 
 
-def client_thread(
-    url: str,
-    gtfs_service_date: date,
-    scheduled_trips: pd.DataFrame,
-    scheduled_stop_times: pd.DataFrame,
-    stops: pd.DataFrame,
-):
+def client_thread(url: str):
     logger.info(f"Connecting to {url}...")
     client = sseclient.SSEClient(requests.get(url, headers=HEADERS, stream=True))
 
     current_stop_state: Dict = disk.read_state()
 
-    process_events(client, current_stop_state, gtfs_service_date, scheduled_trips, scheduled_stop_times, stops)
+    process_events(client, current_stop_state)
 
 
-def process_events(
-    client: sseclient.SSEClient,
-    current_stop_state: dict,
-    gtfs_service_date: date,
-    scheduled_trips: pd.DataFrame,
-    scheduled_stop_times: pd.DataFrame,
-    stops: pd.DataFrame,
-):
+def process_events(client: sseclient.SSEClient, current_stop_state):
```

Comment:
Nitpick, let's type hint as much as possible going forward
```suggestion
def process_events(client: sseclient.SSEClient, current_stop_state: dict):
```